scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -1142,7 +1147,7 @@ def event_slice_optimisation(self):
 
     @event_slice_optimisation.setter
     def event_slice_optimisation(self, value):
-        self.event_slice_optimisation_checkbox.setChecked(value)
+        pass
```

Comment:
It's great to have the comment on line 390 to make sure that a developer looking through the code would understand why these changes have been made. This change is quite a bit further down the file than the comment, though, and I wonder if it could cause confusion for a new developer if they stumbled on this without having seen the comment further up. We could add another comment here, or we could consider just removing this setter for now? I can't see anywhere that it's used and it would be quick to add back in if needed again in future.

Reply:
Yes, that's a good point, removing the setter altogether could be similarly confusing. Adding a comment here sounds good then, thanks.